Modules/Filtering/ImageIntensity/include/itkMaskNegatedImageFilter.h

```diff
@@ -93,13 +93,6 @@ class MaskNegatedInput
     return TPixelType{};
   }
 
-  template <typename TValue>
```

Comment:
This was probably added to support `VariableLengthVector`s. I would not remove it.

Reply:
Restored.